lib/model/place_details.dart

```diff
@@ -21,14 +20,24 @@ class PlaceDetails {
   );
 }
 
+class PhotoClass {
```

Comment:
This class is not needed. You can delete this as it's the same as `class Photo` in `places.dart`